test/test_helper.rb

```diff
@@ -44,5 +44,6 @@ def before_setup
     WebMock.disable_net_connect!
     WebMock.stub_request(:get, "https://app.shopify.com/services/apis.json").to_return(body: API_META_TEST_RESPONSE)
     ShopifyAppConfigurer.call
+    Rails.application.reload_routes!
```

Comment:
I've opted to call this in `ActiveSupport::TestCase#before_setup` to run before each test's setup block. 

I haven't noticed a considerable change in test runtime.

Reply:
This allows us to remove all other instances of `Rails.application.reload_routes!` across our tests